pavelib/pylint.py

```diff
@@ -0,0 +1,126 @@
+from __future__ import print_function
+import os
+import re
+
+from pavelib.utils.cmd import shell
+from pavelib.utils.envs import Env
+
+PACKAGE_DIRECTORIES = [
+    'djangoapps',
+    'lib',
+]
+
+
+def pylint(name, services, *flags):
+    Env.METRICS_DIR.makedirs_p()
+    count = 0
+    for service in services:
+        modules = _get_modules(service)
+        environment = get_python_path(service)
+        directory = get_clean_report_directory(service)
+        report = "{directory}/{name}.report".format(
+            directory=directory,
+            name=name,
+        )
+        shell(
+            environment,
+            'pylint',
+            '--msg-template',
+            "'{path}:{line}: [{msg_id}({symbol}), {obj}] {msg}'",
+            ' '.join(flags),
+            modules,
```

Comment:
Same question here...just want to be sure reports wouldn't get overwritten as directories are crawled.